### PR TITLE
word choice changes

### PR DESCRIPTION
**Pull request by btkostner**
Spent some time reading over the `README.md` and completing the `package.json`.

* Quite a bit of packages are updated
* All public facing instances of 'houston' are now 'Houston'
* Added a `LICENSE.md` (copy from mvp)
* Changed some words around in the `README.md`. Most notably the description, and the hacking area. It is now more clear that a recent version of node is needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,86 @@
...
+If any test fails, Houston will submit an issue on GitHub with detailed
+information on the problems, and in some cases, solutions to fix it. After
+fixing these issues you will be able to resubmit your application for
+publishing.
+
+At this point, the approved application will be moved into the stable repo, and
+available to all users of elementary. We will be using
+[aptly](https://github.com/smira/aptly) behind nginx to host the repository.
+Ultimately this repository will be the focus of
+[AppCenter](https://launchpad.net/appcenter) and the elementary OS's 3rd party
```
Comment (danrabbit) [inline README.md]: remove "the"

File: README.md [hunk 2]
```diff
@@ -1,56 +1,86 @@
...
+application and continue running tests. This ensures the best performance, user
+experience, and desktop integration for any application in AppHub.
+
+If any test fails, Houston will submit an issue on GitHub with detailed
+information on the problems, and in some cases, solutions to fix it. After
+fixing these issues you will be able to resubmit your application for
+publishing.
+
+At this point, the approved application will be moved into the stable repo, and
+available to all users of elementary. We will be using
```
Comment (cassidyjames) [inline README.md]: Change `elementary` to `elementary OS`

File: README.md [hunk 3]
```diff
@@ -1,56 +1,86 @@
...
+[aptly](https://github.com/smira/aptly) behind nginx to host the repository.
+Ultimately this repository will be the focus of
+[AppCenter](https://launchpad.net/appcenter) and elementary OS's 3rd party
```
Comment (cassidyjames) [inline README.md]: Should `elementary OS's` be `the elementary OS`? I'm undecided, but "OS's" can be kind of awkward.

Comment (btkostner) [inline README.md]: I changed it to the elementary OS because you are right and it sounds weird as "OS's"

**Pull request closed without merging**